composer.json

```diff
@@ -23,6 +23,7 @@
             "email": "contact@mikesimonson.com"
         }
     ],
+    "minimum-stability": "beta",
```

Comment:
This can be reverted when Symfony 6.0 is stable.

Reply:
Can you maybe instead try https://github.com/doctrine/.github/pull/29 on this PR? That way if we have a dependency that is not compatible with Symfony 6 yet, we will know.

Reply:
> I'm not sure what's up with the BC check,

That check is drunk. It lists documented BC breaks of Symfony, but those are fine when bumping to a new major.

> but maybe dropping this line will fix it.

It might, but it will likely break again as soon as Symfony 6 is stable.

Reply:
Maybe it's because this package defines Symfony commands, and thus shares some BC-breaks of Symfony?

Reply:
Yes, more precisely:

> [BC] CHANGED: The parameter $default of Symfony\Component\Console\Command\Command#addArgument() changed from no type to a non-contravariant ?mixed

This is not a breaking change, neither for the caller nor for downstream implementations.

> [BC] CHANGED: The return type of Symfony\Component\Console\Command\Command::getDefaultName() changed from no type to ?string

This is technically a BC break for code that extends one of the commands defined in the migrations package and overrides one of the methods listed there. Then again, a package that does something like that is probably aware of symfony/console because the commands are only useful when being integrated in a Symfony console application.

If we really treated this as a BC break, we would need to create a new major release for Symfony 6 support and either disallow Symfony 5 there or decalre all commands `final`. I really don't think that this is necessary.

Reply:
I have switched to your workflow. It installs the correct packages all right and the BC check failure is gone as we expected. But the tests fail with your workflow and they didn't do that with my previous `minimum-stability` hack and neither do they fail when I execute the tests locally. 🤔

Reply:
:eyes:  The failure seems related to the `COLUMNS` env variable

Reply:
Indeed. I've moved the `COLUMNS` env to PHPUnit's config file. That's imho where the setting belongs if the tests fail without it.